power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/newton_raphson_pf_solver.hpp

```diff
@@ -213,7 +213,7 @@ template <bool sym> class NewtonRaphsonPFSolver : public IterativePFSolver<sym,
           perm_(y_bus.size()) {}
 
     // Initilize the unknown variable in polar form
-    void initialize_derived_solver(YBus<sym> const&, MathOutput<sym> const& output) {
+    void initialize_derived_solver(YBus<sym> const& /*unused*/, MathOutput<sym> const& output) {
```

Comment:
```suggestion
    void initialize_derived_solver(YBus<sym> const& /*y_bus*/, MathOutput<sym> const& output) {
```

or again double-check with the other solvers